JavaFXUserInterface/src/main/java/Controllers/SystemTypeController.java

```diff
@@ -61,11 +78,10 @@ public void fillSystemTypeTable() {
      * This method will return an ObservableList of SystemList objects
      *
      * @author Shirwa
-     *
+     * <p>
```

Comment:
why?

Reply:
sorry, no idea what that is, i'll remove it.